### PR TITLE
Fixes rending of the form field in Django 3+

### PR DESCRIPTION
**Pull request by richardARPANET**
Also includes https://github.com/Aristotle-Metadata-Enterprises/django-jsonforms/pull/9

### CODE REVIEW

File: django_jsonforms/forms.py [hunk 1]
```diff
@@ -92,6 +91,9 @@ def clean(self, value):
 
         return value
 
+    def prepare_value(self, value):
+        return json.dumps(value)
```
Comment (richardARPANET) [inline django_jsonforms/forms.py:95]: This method is the bug fix.

**Comment by richardARPANET** [comment]
CI isn't running but the tests do pass:

```
env PYTHONPATH=. env DJANGO_SETTINGS_MODULE=django_jsonforms.tests.testapp.settings django-admin test
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.........sss
----------------------------------------------------------------------
Ran 12 tests in 0.014s

OK (skipped=3)
Destroying test database for alias 'default'...
```

**Comment by harrywhite4** [comment]
Thanks for that, will fix up that ci and get a release going

**Comment by harrywhite4** [comment]
Have just released 1.1.1

**Pull request merged**